extensions.json

```diff
@@ -1049,8 +1048,8 @@
     {
       "id": "timonwong.shellcheck",
       "repository": "https://github.com/timonwong/vscode-shellcheck",
-      "version": "0.10.1",
-      "checkout": "v0.10.1"
+      "version": "0.11.0",
+      "checkout": "master"
```

Comment:
The latest release tag still seems to be `v0.10.1`. Please undo this change.

Reply:
Done!